helix-view/src/editor.rs

```diff
@@ -34,6 +34,43 @@ where
     Ok(Duration::from_millis(millis))
 }
 
+#[derive(Debug, Clone, PartialEq, Deserialize)]
+#[serde(rename_all = "kebab-case", default, deny_unknown_fields)]
+pub struct FilePickerConfig {
+    pub hidden: bool,
+    pub parents: bool,
+    pub ignore: bool,
+    pub git_ignore: bool,
+    pub git_global: bool,
+    pub git_exclude: bool,
+    pub follow_links: bool,
+    pub max_depth: Option<usize>,
+    // filter_entry - maybe use for a handle to block descent into .git ?
+}
+
+impl Default for FilePickerConfig {
+    fn default() -> Self {
+        Self {
+            // could simply use .standard_filters if these are uniform.
+            // Enables ignoring hidden files.
+            hidden: false,
```

Comment:
The defaults should match the `WalkBuilder` defaults.

hidden: true, ignore: true, git-ignore: true, git-global: true, git-exclude: true, follow-links: false, max-depth: None

Reply:
I will set defaults to match the `ignore` crate's in my forthcoming commits.